src/psyclone/dynamo0p3.py

```diff
@@ -4316,7 +4688,8 @@ def function_space(self):
         ''' Returns the expected finite element function space for this
             argument as specified by the kernel argument metadata. '''
         if self._type == "gh_operator":
-            return self.function_spaces[1]
+            # We return the 'from' space for an operator argument
```

Comment:
document args in parent function_space method docstring

Reply:
Done.